packages/tauri/src-tauri/tauri.conf.json

```diff
@@ -75,7 +75,7 @@
             }
         ],
         "security": {
-            "csp": "default-src blob: data: filesystem: ws: wss: http: https: tauri: 'unsafe-eval' 'unsafe-inline' 'self'; style-src 'self' 'unsafe-inline'; object-src 'self' blob:; frame-src 'self'; img-src 'self' blob: data: https:;"
+            "csp": null
```

Comment:
Apparently tauri now automatically adds nonces to inline `<script>` and `<style>` tags to white-list them explicitly, which conflicts with how lit injects style sheets... So I had to disable it altogether. Hopefully in the future they'll offer a way to opt out of the nonce stuff but including in a csp tag in the `index.html` directly works just as well I think (not sure if there are any differences security-wise).